.github/workflows/build-push-supernova-image.yaml

```diff
@@ -0,0 +1,108 @@
+
+      # This action enables you to SIGN and VERIFY container images using cosign
+      # https://github.com/sigstore/cosign-installer
+      - name: Install cosign
+        uses: sigstore/cosign-installer@v3.6.0
+        with:
+          cosign-release: "v2.4.0"
+
+      # Add support for more platforms with QEMU (optional)
+      # QEMU is a generic and open source machine & userspace emulator and virtualizer.
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      # Set up BuildKit Docker container builder to be able to build MULTI-platform images and export cache
+      # https://github.com/docker/setup-buildx-action
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+        with:
+          driver-opts: |
+            image=moby/buildkit:latest
+
+      # Login against a Docker registry except on PR
+      # https://github.com/docker/login-action
+      - name: Log into registry ${{ env.REGISTRY }}
+        if: github.event_name != 'pull_request'
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Read Docker image version
+        id: read_version
+        run: echo "IMAGE_VERSION=$(cat ${{ env.PACKAGE_PATH }}/docker/VERSION)" >> $GITHUB_ENV
+
+      # Extract metadata (tags, labels) for Docker
+      # https://github.com/docker/metadata-action
+      - name: Extract Docker metadata
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: ${{ env.REGISTRY }}/${{ github.repository_owner }}/${{ env.IMAGE_NAME }}
+          tags: |
+            # Use the version from the VERSION file
+            type=raw,value=${{ env.IMAGE_VERSION }},prefix=v
+            # Optionally include semver tags
+            type=semver,pattern={{major}}.{{minor}},prefix=v
+            # Additional Useful Tags
+            type=sha,enable=true,format=short,prefix=
+            type=edge,branch=master
+          labels: |
+            org.opencontainers.image.title=Supernova UI
+            org.opencontainers.image.description=Supernova is an alternative UI for Prometheus Alertmanager with some quality of life improvements
+            org.opencontainers.image.url=https://github.com/cloudoperators/greenhouse-extensions/tree/main/alerts/ui
+
+      # Build and push Docker image with Buildx (don't push on PR)
+      # https://github.com/docker/build-push-action
+      - name: Build and push Docker image
+        id: build-and-push
+        uses: docker/build-push-action@v6
+        with:
+          context: ${{ env.PACKAGE_PATH }}
+          file: ${{ env.PACKAGE_PATH }}/docker/Dockerfile
+          # change to true to push
+          push: false
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+          platforms: |
+            linux/amd64
+            linux/arm64
```

Comment:
I'm missing the privy check from the gh-action like this:
https://github.com/cloudoperators/greenhouse/blob/cd0e226dc8e272106d15f05b60d37667a2e7dc0a/.github/workflows/docker-build.yaml#L138-L180

Reply:
This is planed for the second iteration. The build and push step is for now deactivated. We want just see the action in action :)